File: AppRemover/src/main/java/com/vs_unusedappremover/PackagesChangedReceiver.java
package com.vs_unusedappremover;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class PackagesChangedReceiver extends BroadcastReceiver {

	@Override
	public void onReceive(Context context, Intent intent) {
		Log.i(MyApplication.TAG, "received " + intent.getAction());
		MyApplication.getInstance().getApplications().notifyChanged();
	}
}

File: AppRemover/src/main/java/com/vs_unusedappremover/PackagesChangedReceiver.kt
package com.vs_unusedappremover

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class PackagesChangedReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        Log.i(MyApplication.TAG, "received " + intent.action)
        MyApplication.instance.applications.notifyChanged()
    }
}
